File: Data Structures & Algorithms/longest-common-prefix/submission-0.ts
class Solution {
    /**
     * @param {string[]} strs
     * @return {string}
     */
    longestCommonPrefix(strs: string[]): string {
        let ans: string = "";
        for (let i = 0; i < strs[0].length; i++) {
            for (let j = 1; j < strs.length; j++) {
                if (strs[j][i] !== strs[0][i]) return ans;
            }
            ans = `${ans}${strs[0][i]}`;
        }
        return ans;
    }
}

File: Data Structures & Algorithms/validate-parentheses/submission-0.py
class Solution:
    def isValid(self, s: str) -> bool:
        ss = ""
        for c in s:
            if ss and ((c == ")" and ss[-1] == "(") or
                        (c == "]" and ss[-1] == "[") or
                        (c == "}" and ss[-1] == "{")):
                ss = ss[:-1]
            else:
                ss += c
        return not ss